README.md

```diff
@@ -1,12 +1,15 @@
 # OMP OPC UA Edge Connector 
 ### **Software under construction**
 
-- [Motivation](#motivation)
-- [Infrastructural View](#infrastructural-view)
-- [Application View](#application-view)
-- [Operational View](#operational-view)
-- [Getting Started and Contribution](#getting-started-and-contribution)
-- [Build and Installation](#build-and-installation)
+- [OMP OPC UA Edge Connector](#omp-opc-ua-edge-connector)
```

Comment:
Looks like you duplicated the rows that are displayed as headings.

Was this intentional?

![image](https://user-images.githubusercontent.com/88766457/163960997-147ea336-872a-4ec9-9db0-a7de229d36e1.png)

Reply:
Thanks, I have resolved the issue. It's a result of a merge conflict